department-of-veterans-affairs/caseflow
# Changed Pexip pin length

Resolves #13856 

### Description
Please explain the changes you made here.

### Acceptance Criteria

### Testing Plan
1. Select judge "BVAYELLOW": http://localhost:3000/test/users
1. Navigate to hearings:  http://localhost:3000/hearings/schedule
1. Choose a "Video" hearing
1. Click the "Edit hearing details" link
1. Change from a "Video" hearing to a "Virtual" hearing
1. Once the virtual hearing has been scheduled, confirm that the host pin is 7 characters long and the guest pin is 10 characters long

### User Facing Changes

 BEFORE|AFTER
 ---|---
![BeforeIncreasePinLength](https://user-images.githubusercontent.com/61989526/81735202-af7afc00-9462-11ea-99ff-1d8dc2c2b9e5.png)|![AfterIncreasePinLength](https://user-images.githubusercontent.com/61989526/81734884-34b1e100-9462-11ea-9bed-cec14f833d3b.png)

### Database Changes
*Only for Schema Changes*

* [x] DB schema docs updated with `make docs`
